carcara/src/checker/rules/pb_blasting.rs

```
@@ -574,3 +574,39 @@ pub fn pbblast_bvand(RuleArgs { pool, conclusion, .. }: RuleArgs) -> RuleResult
 
     Ok(())
 }
+
+/// This rule extracts assertions of the ith bit of an application of
+/// `pbblast_bvand`, given its arguments were x and y, we conclude
+///  `(>= x r) (>= y r) (>= (+ r 1) (+ x y)))`
+/// In which ri is the choice element from the pseudo boolean bit blasting
+/// of the bvand rule
+pub fn pbblast_bvand_ith_bit(RuleArgs { args, pool, conclusion, .. }: RuleArgs) -> RuleResult {
+    let x = &args[0];
+    let y = &args[1];
```

You should make sure that exactly two arguments are passed, using `super::assert_num_args(args, 2)?;`. Currently, if no argument is passed to the rule, it will panic instead of returning an error.

---

Good catch! I implemented the fix and some unit tests to catch it!